fetch_raw_data.py

```diff
@@ -0,0 +1,247 @@
+import argparse
+import csv
+import json
+import os
+from datetime import datetime
+from io import StringIO
+
+import pytz
+from core_data_modules.cleaners import Codes, PhoneCleaner
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data import Metadata, TracedData
+from core_data_modules.traced_data.io import TracedDataJsonIO
+from core_data_modules.util import IOUtils, TimeUtils, SHAUtils
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from rapid_pro_tools.rapid_pro_client import RapidProClient
+from storage.google_cloud import google_cloud_utils
+from temba_client.v2 import Contact, Run
+
+from src.lib import PipelineConfiguration, CodeSchemes
+from src.lib.pipeline_configuration import RapidProSource, GCloudBucketSource, ShaqadoonCSVSource
+
+log = Logger(__name__)
+
+
+def label_somalia_operator(user, traced_runs, phone_number_uuid_table):
```

Comment:
Not quite sure this is necessary for this project, but I could be wrong.

Reply:
Thanks for reviewing so thoroughly.

This is definitely not necessary. I had planned to copy what I thought was standard and then learn what broke, but glad you picked this up so quickly. Ultimately this needs to go into configuration somewhere rather than being copied or not copied depending on the project, but given COVID19's timelines I will remove for now and add issues against both this repo and IOM (filing an issue against IOM because that's where pipeline innovation is currently happening, and where I expect the next project will be copied from)

Reply:
https://github.com/AfricasVoices/Project-IOM/issues/49